models/index.js

```diff
@@ -1,18 +1,58 @@
-const booking = require('./booking');
-const industry = require('./industry');
-const services = require('./service');
-const user = require('./users');
+const Booking = require('./booking');
+const Industry = require('./industry');
+const Service = require('./service');
+const User = require('./user');
+const UserIndustry = require('./userIndustry');
 
+User.hasMany(Service, {
+  foreignKey: 'id',
+  onDelete: 'CASCADE',
+});
 
-      
+Service.belongsTo(User, {
+  foreignKey: 'id',
+});
 
-user.hasMany(services, {
-    foreignKey: 'id',
-    onDelete: 'CASCADE',
-  });
-  
-booking.belongsTo(users, {
-    foreignKey: 'id',
-  });
+//one user has many bookings
+User.hasMany(Booking, {
+  foreignKey: 'id',
+  onDelete: 'CASCADE',
+});
 
-module.exports = { , , };
+Booking.belongsTo(User, {
+  foreignKey: 'id',
+});
+
+//one booking to one user
+Booking.hasOne(User, {
+  foreignKey: 'id',
+});
+
+User.belongsTo(Booking, {
+  foreignKey: 'id',
+});
+
+//one booking to one service
+Booking.hasOne(Service, {
+  foreignKey: 'id',
+});
+
+Service.belongsTo(Booking, {
+  foreignKey: 'id',
+});
+//one service can have many booking
+Service.hasMany(Booking, {
+  foreignKey: 'id',
+  onDelete: 'CASCADE',
+});
+
+//many users to many industry
+users.belongsToMany(Industry, {
```

Comment:
Please review the name of this variable reference and please additionally review how to resolve many to many relationships as per docs: https://sequelize.org/docs/v6/advanced-association-concepts/advanced-many-to-many/

You can also look at project 13, the E-Commerce backend, for a decent example of this